server/build.gradle.kts

```diff
@@ -75,6 +75,7 @@ tasks.register<GenerateTask>("openapiGenerateDeltaSharing") {
 }
 
 tasks.withType<Test> {
+    forkEvery = 1
```

Comment:
maybe we can do this only for delta tests? Since this will slow down the build a lot